HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRPlayer.cs

```diff
@@ -434,7 +434,9 @@ private void GetPlayerDebuffAbnormalities()
             MHRDebuffStructure abnormality = new();
 
             // Only read memory if the required sub Id is the required one for this abnormality
-            if (abnormSubId == schema.WithValue)
+            if (schema.IsInfinite)
+                abnormality.Timer = (schema.CompareOpr == (int)CompareType.WithValue && abnormSubId == schema.WithValue) || (schema.CompareOpr == (int)CompareType.WithValueNot && abnormSubId != schema.WithValueNot) ? AbnormalityData.TIMER_MULTIPLIER : 0;
+            else if ((schema.CompareOpr == (int)CompareType.WithValue && abnormSubId == schema.WithValue) || (schema.CompareOpr == (int)CompareType.WithValueNot && abnormSubId != schema.WithValueNot))
```

Comment:
When I said that I dislike nesting if/elses, I didn't mean that you should inline everything, that's unreadable...